sdk/tables/azure-data-tables/tests/test_table_client.py

```diff
@@ -500,7 +496,6 @@ def test_create_service_with_token(self):
         assert service.url.startswith('https://' + self.tables_storage_account_name + suffix)
         assert service.credential == self.token_credential
         assert not hasattr(service.credential, 'account_key')
-        assert hasattr(service.credential, 'get_token')
```

Comment:
I'm kinda not sure how this line ever passed....

Reply:
Because these were unit tests we used a fake credential the whole time. I think these were among the tests copied from storage too...